docs/data.rst

```diff
@@ -196,6 +196,9 @@ with numeric fields, returning the number as a string. For example:
     contains a value that can be interpreted as a number, ``sample["my_string"]``
     returns a number, not a string.
 
+
+.. _section-access-64-bit-integers:
+
 Accessing 64-bit integers
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 Internally, *Connector* relies on a framework that only contains a single number
```

Comment:
@samuelraeburn, remember in this file somewhere, you were going to remove a Warning and add it to somewhere in the release notes: https://github.com/rticommunity/rticonnextdds-connector-py/pull/102#discussion_r751983361. Feel free to update this branch directly.

Reply:
thanks for the reminder!